packages/itwinui-react/src/core/ExpandableBlock/ExpandableBlock.tsx

```diff
@@ -127,10 +140,12 @@ export const ExpandableBlock = React.forwardRef((props, ref) => {
       {...rest}
     >
       <Box
+        role='button'
         aria-expanded={expanded}
         className='iui-header'
+        aria-disabled={disabled}
         tabIndex={0}
-        onClick={handleToggle}
+        onClick={() => handleToggle()}
```

Comment:
It was left over from when I was moving the disabled check from onClick to the handleToggle function. 

I cleaned up the line so that it just calls {handleToggle} like in the original version.